sorting/python/quick-sort.py

```diff
@@ -0,0 +1,12 @@
+def quickSort(arr):
+    n = len(arr)
+    if n == 1 or n == 0:
+        return arr
+    pi = 0
```

Comment:
asking just out of curiosity 🙃
why did you went for `first point pivot` technique ?

although there is nothing wrong in it, but just asking if there was a specific reason or you just found it easier to implement it this way.

Reply:
I just found it easier. I implemented it when I was first studying DSA.